app/controllers/matches_controller.rb

```diff
@@ -3,6 +3,7 @@
 class MatchesController < ApplicationController
   def index
     active_tab
+    @date_var = Date.new
```

Comment:
I think you do not need to define a instance variable here, you can do it in the view partial instead as a normal ruby variable. Only use instance variables if you want to pass information from controller to view. Since I can do `Date.new` anywhere, no need to do this in controller.